code_style.sh

```diff
@@ -40,11 +40,10 @@ find_cfiles() {
        ! \( $(get_find_path_args $(trim_comments .clang-format-ignore)) \) "$@"
 }
 
-# We're targeting clang-format version 17 and other versions give slightly
-# different results, so prefer `clang-format-17` if it's installed.
+# We're targeting clang-format version 17 and other versions give
+# different results, so only accept `clang-format-17` if it's installed.
 clang_format_exe_names=(
   'clang-format-17'
-  'clang-format'
```

Comment:
This wouldn't work for Fedora users. Fedora doesn't provide the `clang-format-${version}` alias.

Reply:
That's not the point here: clang-format-17 is mostly unavailable on macOS as well.
We just want to **disable** clang-format for now because clang-format-17 is not compatible with clang-format-20 output and the pull request to address that is NOT being reviewed/merged fast enough.

If you were merging https://github.com/transmission/transmission/pull/7499 quickly then no need to have this change. It's just a measure to unblock the situation faster for macOS users who are always on latest clang-format.

Reply:
I found how to deal with Fedora using `clang-format --version`: e2d51af0a04f2c85d10cb13fc37ef54489dc2768